bump-version/index.ts

```diff
@@ -0,0 +1,80 @@
+import { chdir, cwd } from 'process'
+import { join } from 'path'
+import { exec } from 'child_process'
+import { promisify } from 'util'
+
+import { getInput, setFailed, info, debug } from '@actions/core'
+import { inc, minor, patch, prerelease, ReleaseType } from 'semver'
+import { readJSON } from 'fs-extra'
+
+import getConventionalBump from './get-conventional-bump'
+
+const asyncExec = promisify( exec )
+
+type Package = {
+  version: string,
+}
+
+const nextInc = ( version: string, releaseType: string ) => inc( version, releaseType as ReleaseType, 'next' ) ?? version
+const removeNext = ( version: string ) => nextInc( version, 'patch' )
+
+const increment = ( version: string, releaseType: ReleaseType, isNext: boolean ) => {
+  // Current version is a next release or not
+  const currentNext = !!prerelease( version )
```

Comment:
Do you think we could make a coding convention to begin bool variables with "is"? So `isCurrentlyNext` instead of `currentNext`

Reply:
`is` or `has` prefix, but i don't think we can enforce it with tooling btw

Reply:
Would this work?

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md#enforce-that-boolean-variables-are-prefixed-with-an-allowed-verb

Reply:
That is perfect. Please PR in?

Reply:
To actions?